install.sh

```diff
@@ -0,0 +1,9 @@
+os=$([ $(uname | grep 'Darwin') ] && echo 'darwin' || echo 'linux');
+curl -s https://api.github.com/repos/livepeer/go-livepeer/releases/tags/0.1.7 \
+  | grep browser_download_url \
+  | grep "${os}.tar" \
+  | cut -d '"' -f 4 \
+  | xargs -n 1 curl -L \
+  | tar xz \
+  && cp -R "livepeer_${os}/." /usr/local/bin \
```

Comment:
Should we also rename `livepeer_darwin` to just `livepeer` while we're doing this?

Reply:
I don't think it matters much since that directory gets deleted on the next line anyways.

Reply:
ohhh, i was thinking the binaries themselves rather than the directory. They used to be named `livepeer_darwin`, etc, but since they're not, we're all good.